### DISCUSSION TITLE
CPU性能调节的疑问

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#87] [Q&A/答疑解惑] [answered]
新编的ax6固件，cpu频率默认是在1—1.4之间浮动，在cpu性能调节中超频到1.7之后，观察许久发现频率在1.4—1.7之间浮动，不会像之前一样降到1，请问是bug还是原本就是这样设计的。

**Comment by Boos4721** [comment, marked as answer]
governor issue.

**Reply by ghost** [reply]
在netdata里看了下其实cpu频率调节是正常的，应该只是总览页面显示的bug

**Reply by Boos4721** [reply]
> 在netdata里看了下其实cpu频率调节是正常的，应该只是总览页面显示的bug

KK

**Reply by zjwxh** [reply]
AX6 cpu超频到2.2G后貌似跑分和不超频没啥区别，这个频率只是数字显示没有实质提升吗 ？

**Comment by longge1120** [comment]
ax3600超频后的coremark跑分和没超的一样，超频只是数字显示有区别嘛？
